nodebuilder/tests/tastora/framework.go

```diff
@@ -275,76 +293,69 @@ func (f *Framework) fundNodeAccount(ctx context.Context, daNode *tastoradockerty
-					Version:    getCelestiaTag(),
-					UIDGID:     "10001:10001",
-				},
-			},
-			Bin:            "celestia-appd",
-			Bech32Prefix:   "celestia",
-			Denom:          "utia",
-			CoinType:       "118",
-			GasPrices:      "0.025utia",
-			GasAdjustment:  1.3,
-			EncodingConfig: &enc,
-			AdditionalStartArgs: []string{
-				"--force-no-bbr",
-				"--grpc.enable",
-				"--grpc.address", "0.0.0.0:9090",
-				"--rpc.grpc_laddr", "tcp://0.0.0.0:9098",
-				"--timeout-commit", "1s",
-			},
-		},
-		DataAvailabilityNetworkConfig: &tastoradockertypes.DataAvailabilityNetworkConfig{
-			BridgeNodeCount: cfg.BridgeNodeCount,
-			LightNodeCount:  cfg.LightNodeCount,
-			Image: tastoradockertypes.DockerImage{
-				Repository: getNodeImage(),
-				Version:    getNodeTag(),
-				UIDGID:     "10001:10001",
-			},
-		},
+	// Create chain builder
+	chainImage := container.Image{
+		Repository: celestiaAppImage,
+		Version:    getCelestiaTag(),
+		UIDGID:     "10001:10001",
 	}
-	return tastoradockertypes.NewProvider(dockerCfg, f.t)
-}
 
-// createAndStartCelestiaChain initializes and starts the Celestia chain.
-func (f *Framework) createAndStartCelestiaChain(ctx context.Context) tastoratypes.Chain {
-	celestia, err := f.provider.GetChain(ctx)
-	require.NoError(f.t, err, "failed to get chain")
+	chainBuilder := cosmos.NewChainBuilderWithTestName(f.t, f.t.Name()).
+		WithDockerClient(f.client).
+		WithDockerNetworkID(f.network).
+		WithImage(chainImage).
+		WithEncodingConfig(&enc).
+		WithAdditionalStartArgs(
+			"--force-no-bbr",
+			"--grpc.enable",
+			"--grpc.address", "0.0.0.0:9090",
+			"--rpc.grpc_laddr", "tcp://0.0.0.0:9098",
+			"--timeout-commit", "1s",
+		).
+		WithPostInit(func(ctx context.Context, node *cosmos.ChainNode) error {
+			if err := config.Modify(ctx, node, "config/config.toml", func(cfg *cometcfg.Config) {
+				cfg.TxIndex.Indexer = "kv"
+			}); err != nil {
+				return err
+			}
+			return config.Modify(ctx, node, "config/app.toml", func(cfg *servercfg.Config) {
+				cfg.GRPC.Enable = true
+			})
+		})
+
+	// Add validator nodes based on config
+	for i := 0; i < cfg.NumValidators; i++ {
+		nodeConfig := cosmos.NewChainNodeConfigBuilder().Build()
+		chainBuilder = chainBuilder.WithNode(nodeConfig)
+	}
 
-	err = celestia.Start(ctx)
-	require.NoError(f.t, err)
+	// Create DA network builder with just one bridge node for now
+	daImage := container.Image{
+		Repository: getNodeImage(),
+		Version:    getNodeTag(),
+		UIDGID:     "10001:10001",
+	}
+
+	// always have at least one bridge node.
+	bridgeNodeConfig := dataavailability.NewNodeBuilder().
+		WithNodeType(types.BridgeNode).
+		Build()
 
-	// verify the chain is producing blocks
-	require.NoError(f.t, wait.ForBlocks(ctx, 2, celestia))
-	return celestia
+	daNetworkBuilder := dataavailability.NewNetworkBuilderWithTestName(f.t, f.t.Name()).
+		WithDockerClient(f.client).
+		WithDockerNetworkID(f.network).
+		WithImage(daImage).
+		WithNodes(bridgeNodeConfig)
```

Comment:
did we remove the static topology and we no longer need the bridge and light node counts to be passed over?